6-Typescript/common/interfaces/DB.ts

```diff
@@ -0,0 +1,11 @@
+import Item from './Item'
+import Guid from '../types/Guid'
+import UserID from '../types/userID'
+export default interface DB {
+    connect(): Promise<boolean>
+    disconnect(): Promise<boolean>
+    getAllItemsFromDB(userID: UserID): Promise<Item[]>
+    getItemFromDB(id: Guid): Promise<Item>
+    setItemInDB(item: Item): Promise<Item>
+    removeItemFromDB(itemID: Guid): Promise<Item>
```

Comment:
why `getItemFromDB` gets `id` and `removeItemFromDB` gets `itemID`?

Reply:
missed it, I'll change it to id